pyrealm_build_data/synth_data.py

```diff
@@ -0,0 +1,81 @@
+"""This script uses a parametrized model to compress the input dataset.
+
+It fits a time series model to the input data and stores the model parameters.
+The dataset can then be reconstructed from the model parameters using the `reconstruct`
+function, provided with a custom time index.
+"""
+from typing import Tuple
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+VAR_BOUNDS = dict(
+    temp=(-25, 80),
+    patm=(3e4, 11e4),
+    vpd=(0, 1e4),
+    co2=(0, 1e3),
+    fapar=(0, 1),
+    ppfd=(0, 1e4),
+)
```

Comment:
Why are these a global variable if they are only used in one function.

Reply:
Good point - I can move it into the function. However, is it worth considering to create a global config of variable bounds over the whole `pyrealm` package, stored somewhere like in `pyrealm/constants`?